tokio-util/tests/stream_reader.rs

```diff
@@ -2,13 +2,14 @@
 #![cfg(feature = "full")]
 
```

Comment:
Should this test file be named `io_stream_reader.rs` for consistency with the `io_reader_stream.rs` test?